napari/layers/_shapes_layer/shape_list.py

```diff
@@ -657,3 +657,36 @@ def to_labels(self, labels_shape=None, shape_type=None):
                     labels[mask] = index[ind]
 
         return labels
+
+    def to_xml(self, shape_type=None):
+        """Returns a list of xml elements defining each shape according to the
```

Comment:
Refrain from using "return" and 3rd-person verb forms (e.g. use 'create' instead of 'creates')

Reply:
fixed